Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -155,18 +155,18 @@ public class SessionController : BaseJellyfinApiController
         /// <param name="startPositionTicks">The starting position of the first item.</param>
         /// <response code="204">Instruction sent to session.</response>
         /// <returns>A <see cref="NoContentResult"/>.</returns>
-        [HttpPost("Sessions/{sessionId}/Playing/{command}")]
```

Comment:
Why did you swap where the route parameter is?
This function has `[FromRoute, Required] PlayCommand command`, the other does not.

Reply:
Play and PlayStateCommand ended up with identical urls - Sessions/{sessionId}/Playing/. The Play method didn't support the {command}, and from what I've read you cannot differentiate based upon payload. 

I moved the {command} from the original play - whilst I tested, as I assumed this is where is actually belonged.

Reply:
This change doesn't really make any sense.

If you want to differentiate between the two endpoints it should be changed to `[HttpPost("Sessions/{sessionId}/Playing/{command:required}")]`, so requests without the `{command}` are properly routed to `SendPlaystateCommand`.